### ISSUE TITLE
Sous-titres

### ISSUE THREAD

**Issue by perive** [Kodi-vStream/venom-xbmc-addons#2224]
Bonjour,

Les sous-titres ne chargent plus sur uptobox (via ZT ou DDL-Island).
C'est depuis hier, même avant la dernière mise à jour.

**Comment by Ovni-crea** [comment]
J'ai tester le film the third murder sur zt en web-dl 1080p j'ai démarré le film et j'ai les sous-titres donc je crois pas que c'est uptobox

**Comment by perive** [comment]
T'as testé aujourd'hui ? Parce que là, ça marche pas chez moi...

**Comment by Arias800** [comment]
Tu a activer dans les paramètres de vstream l'activation automatique des sous titres ?

**Comment by perive** [comment]
yes

**Comment by Ovni-crea** [comment]
@perive oui je venait de tester justement mais ta peut être dépasser le quota

**Comment by perive** [comment]
A noter que vostfr ne se trouve pas dans le titre. Mais avant, ça marchait...

**Comment by Ovni-crea** [comment]
Après c'est peut être le site qui mette pas de sous titres

**Comment by perive** [comment]
Non, j'ai pas les sous-titres pour ton film. Bizarre !

**Comment by Ovni-crea** [comment]
Bon j'en conclue qu'il y a réellement quelque chose pour les sous titres uptobox m'en détecte que 1 c'est english mais vue que j'ai un débrideur j'ai testé les autres et il y a pas english et j'en ai plus comme exemple j'avais pris avengers infinity war et j'avais 2 sous titres que uptobox 1 et c'est pas vstream c'est uptobox eux même car j'avait enlevait le debrideur pour tester en gratuit et c'est pareil donc sa vient pas de vstream mais uptobox(sachant que mon debrideur est hors-vstream add-on externe)

**Comment by perive** [comment]
Oui, j'imagine bien que c'est uptobox car le code de vstream n'a pas changé entretemps.

**Comment by Ovni-crea** [comment]
Si mais bon il ont pas changé la façon de lire les vidéos uptobox mais de toute façon uptobox on plein de problèmes genre avec les débits les comptes etc..donc ça m'étonne pas

**Comment by Ovni-crea** [comment]
Ouais après c'est le seul hébergeur qui a sa pour les sous titres les autres ont rien

**Comment by perive** [comment]
ben, ton film, il est bien en multi, non ?

**Comment by Ovni-crea** [comment]
Si pk

**Comment by perive** [comment]
Il est bien sous-titré et en multi ?

**Comment by Ovni-crea** [comment]
Oui il y a les sous titre french full et french forced

**Comment by Ovni-crea** [comment]
Tu vas sur zt

**Comment by perive** [comment]
0.6.5

**Comment by perive** [comment]
oui j'ai la 0.6.54 (indiqué dans paramètres, dans information c'est écrit 0.6.5)

**Comment by Arias800** [comment]
Fix dans la beta il faut mettre a jour le fichier upto_ox.py (https://raw.githubusercontent.com/Kodi-vStream/venom-xbmc-addons/Beta/plugin.video.vstream/resources/hosters/uptobox.py)
Et uptostr_am.py (https://raw.githubusercontent.com/Kodi-vStream/venom-xbmc-addons/Beta/plugin.video.vstream/resources/hosters/uptostream.py ) ^^

**Comment by perive** [comment]
ça marche, merci Arias800

**Comment by Arias800** [comment]
Oui c'est parce que le nom du fichier sur Uptobox ne contient ni vostfr ni multi donc vstream n'active pas les sous titre ^^

**Comment by perive** [comment]
Y'a pourtant multi qui apparait... en apparence de la même façon que les autres...
Il faudrait pouvoir activer les sous-titres manuellement,

**Comment by Arias800** [comment]
Oui mais ça c'est le nom afficher par zone telechargement pas par uptobox mais je pense que je vais forcer les sous titre si il n'y a pas French ou TrueFrench comme ça pas de probleme ^^

**Comment by Arias800** [comment]
C'est bon fichier mis a jour normalement comme ca c'est bon ^^

**Comment by perive** [comment]
Bonne idée ! Surtout qu'on peut désactiver les st mais pas les activer... Donc, vaut mieux trop que pas assez ;)
C'est donc ok, merci.
